src/install/install.zig

```diff
@@ -1679,6 +1683,10 @@ pub const PackageManager = struct {
         return this.env.getHttpProxy(url);
     }
 
+    pub fn tlsRejectUnauthorized(this: *PackageManager) bool {
+        return this.env.getTLSRejectUnauthorized();
```

Comment:
probably worth storing as a property on package_manager since it will be called a large number of times